app/javascript/components/taggingWrapper.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
```

Comment:
In react 16.4+ some life cycle methods are removed and marked as unsafe (componentWillMount is one of them). Move the initialization code to `componentDidMount` and add some spinner while the state is loading.

Also we can use `connect` from react-redux; Please try to remove `ManageIQ.redux` references and use mapStateToProps/mapDispatchToProps. Here is example how you could do this.
```
mapDipspatchToProps = dispatch => ({
	initialize: dispaych({type: ''}),
	loadState: dispatch({ initialState: this.props.tags, type: "UI-COMPONENTS_TAGGING_LOAD_STATE" }),
	reset: dispatch({ type: "UI-COMPONENTS_TAGGING_RESET_STATE" })
})
mapStateToProps = ({ taggingApp }) => ({
	isLoaded: !!taggingApp,
})
this.props.initialize()
this.props.isLoaded
```